pages/bottommenu.rb

```diff
@@ -0,0 +1,51 @@
+class Bottommenu < BasePage
+
+    def women
+      browser.find_element(name: 'Women')
+    end
+  
+    def Specials
```

Comment:
methods should be named only in lowercase
Specials -> specials
Space character not allowed in methods
New products -> new_products

Reply:
done